### DISCUSSION TITLE
Welcome to freegpt-webui Discussions!

### DISCUSSION THREAD

**Discussion by ramon-victor** [ramonvc/freegpt-webui#74] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  
🔸 Share doubts and ideas for the project.
🔸 Share free API providers and integration models.
🔸 Contribute positively to other community members.
🔸 Welcome others and are open-minded.

  Remember that this is a community we build together 💪.

**Comment by brown-sussy** [comment]
Wow!

**Comment by ibeswipin** [comment]
howdy

**Comment by avesaxe** [comment]
what mean "Evil" ?

**Comment by HaroldObasi** [comment]
@ramonvc still plan on building the frontend with React ?

**Reply by ramon-victor** [reply]
Yes, but no time for now

**Comment by icolec** [comment]
Why does my Discord fail to enter the server and accept invitations

**Comment by icolec** [comment]
Can you tell me? Thank you. I got the api a few days ago and it is not available now

**Comment by SergeiNaum** [comment]
I don't think there's any gpt-4

**Reply by SergeiNaum** [reply]
but this model's okay, too.

**Comment by khitenkov-v** [comment]
hi all. i`m use windows-11 and use VM VirtualBox.
please help me error:
![image](https://github.com/ramonvc/freegpt-webui/assets/141460097/58551ce4-292a-4001-a31b-ac4a504f0ce9)
thanks

**Comment by Thefourthbro** [comment]
I am trying to incorporate this into a project, but it keeps asking for an API key, any suggestions?
